content/en/docs/setup/install/multicluster/single-network/initial-configuration/index.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Initial Configuration
+description: Describes how to complete the initial configuration for a multicluster mesh on a flat network.
+weight: 10
+keywords: [multicluster,first-cluster]
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+To build a {{< gloss >}}multicluster{{< /gloss >}} deployment, configure a
+single cluster {{< gloss >}}service mesh{{< /gloss >}} as your starting point.
+
+The single cluster mesh you deploy in this section, serves as a model for the
+configuration of the first {{< gloss >}}primary cluster{{< /gloss >}}. You need
+at least one primary cluster to complete the [different multicluster deployments.](/docs/ops/deployment/deployment-models/#multiple-clusters)
+
+Each cluster in a multicluster deployment needs a unique identifier. Using
+`CLUSTER_1` as a unique identifier for the first cluster, the following
+diagram shows a service mesh spanning a single primary cluster.
+
+{{< image width="75%"
+    link="init-conf.svg"
+    caption="A service mesh with a single primary cluster"
+    >}}
+
+To set up a cluster as the primary cluster of a multicluster service
+mesh, complete the following configuration steps:
+
+1. [Configure trust for the cluster](#configure-trust)
+1. [Deploy Istio to the cluster](#deploy-istio)
+
+Complete the [Before you begin instructions](/docs/setup/install/multicluster/#before-you-begin)
+before you continue.
```

Comment:
This order is odd.  You say "here are two things that are on this page, which you can see the first one of below the fold, but before any of that - go back to a page you've already been on, but listed as the third step".

(I wonder if there could be a much easier flow through all these pages.)

Reply:
+1. This should probably either be done at the top of the page or removed.

Reply:
Next update moves it a little further up to avoid the issue, thanks.